hil/api.py

```diff
@@ -1131,27 +1131,31 @@ def port_revert(switch, port):
 
     db.session.add(action)
     db.session.commit()
-    return json.dumps({'status_id': unique_id})
+    return json.dumps({'status_id': unique_id}), 202
 
 
-# is this only going to be for statuses of Networking actions? What about in
-# the future we have some other calls whose status we want to check?
-@rest_call('GET', '/status/<status_id>', Schema({'status_id': basestring}))
-def get_status(status_id):
+@rest_call('GET', '/networking_action/<status_id>', Schema({
+    'status_id': basestring}))
+def show_networking_action(status_id):
     """Returns the status of the networking action by findind the status_id
-    in the networking actions table
+    in the networking actions table.
     """
-    # is auth required here? UUID is obscure enough that nobody can guess
-    # and see someone else's status of their operations. Moreover, it doesn't
-    # divulge any secret information about the networking action.
     action = model.NetworkingAction.query.filter_by(uuid=status_id).first()
     if action is None:
         raise errors.NotFoundError('status_id not found')
 
     project = action.nic.owner.project
     get_auth_backend().require_project_access(project)
 
-    return json.dumps({'status': action.status})
+    action_info = {'status': action.status,
+                   'node': action.nic.owner.label,
+                   'nic': action.nic.label,
+                   'type': action.type,
+                   'channel': action.channel,
+                   'new_network': None if action.new_network is None
+                   else action.new_network.label}
```

Comment:
The formatting of the if expression is a bit hard to read; the else should probably be indented a bit further at a minimum (otherwise it reads as being at the same level in the AST as the dictionary keys, as opposed to beneath them).

I also personally find python's syntax for expression-ifs really hard to read in general, but won't block over it.

Reply:
yeah, but I can't place it anywhere otherwise pycodestyl(pep8) complains that it is over-indented.